src/main/java/org/folio/rest/impl/AutomatedPatronBlocksAPI.java

```diff
@@ -50,18 +60,28 @@ public void postAutomatedPatronBlocksSynchronizationJob(SynchronizationJob reque
     Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
     Context vertxContext) {
 
+    log.debug("postAutomatedPatronBlocksSynchronizationJob:: parameters request: {}, okapiHeaders: {}",
+      () -> logAsJson(request), () -> logOkapiHeaders(okapiHeaders));
+
+    Handler<AsyncResult<Response>> loggingResponseHandler =
+      loggingResponseHandler("postAutomatedPatronBlocksSynchronizationJob", asyncResultHandler, log);
+
     new SynchronizationJobService(okapiHeaders, vertxContext.owner())
       .createSynchronizationJob(request)
-      .onSuccess(response -> asyncResultHandler.handle(succeededFuture(
+      .onSuccess(response -> loggingResponseHandler.handle(succeededFuture(
         PostAutomatedPatronBlocksSynchronizationJobResponse
           .respond201WithApplicationJson(response))))
       .onFailure(throwable -> {
         String errorMessage = throwable.getLocalizedMessage();
+        log.warn("postAutomatedPatronBlocksSynchronizationJob:: Failed to create synchronization job",
+          throwable);
         if (throwable instanceof UserIdNotFoundException) {
+          log.debug("postAutomatedPatronBlocksSynchronizationJob:: User ID not found");
```

Comment:
log warn?

Reply:
This logging was removed as redundant (there was a similar comment here). We know which kind of exception is thrown (it's logged a couple of lines above) and we also know which response code is returned - it's logged separately by the web server.